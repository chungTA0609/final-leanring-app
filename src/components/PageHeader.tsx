const PageHeader = () => {
  return (
    <>
      <div className="row">
        <div className="col-12">
          <div
            className={"page-title-box"}
            // :className="{
            //   'page-title-box-alt':
            //     layout.layoutType !== 'vertical' &&
            //     layout.layoutType !== 'two-column',
            // }"
          >
            <h4 className="page-title"> title</h4>
            <div className="page-title-right"></div>
          </div>
        </div>
      </div>
    </>
  );
};
export default PageHeader;
