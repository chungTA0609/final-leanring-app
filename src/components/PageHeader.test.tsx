import renderer from "react-test-renderer";
import PageHeader from "./PageHeader";

it("renders correctly", () => {
  const tree = renderer.create(<PageHeader></PageHeader>).toJSON();
  expect(tree).toMatchSnapshot();
});
