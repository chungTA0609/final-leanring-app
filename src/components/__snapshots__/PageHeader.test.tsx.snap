// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <div
    className="col-12"
  >
    <div
      className="page-title-box"
    >
      <h4
        className="page-title"
      >
         title
      </h4>
      <div
        className="page-title-right"
      />
    </div>
  </div>
</div>
`;
